rss_pipeline/test_load.py

```diff
@@ -0,0 +1,46 @@
+# pylint: skip-file
+from unittest.mock import MagicMock, patch
+from load import *
```

Comment:
Using `import *` should be avoided as per pep8 requirements: https://peps.python.org/pep-0008/#imports